recipes/tpi/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "tpi" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5e7dc5262d81f82dfe3def1f791769ed6024c0791ee52edc841906796dd01778
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - tpi = tpi.main:main
+
+requirements:
+  host:
+    - python >=3.4
+    - pip
+    - setuptools_scm >=3.4
+    - toml
+  run:
+    - python >=3.4
```

Comment:
Upstream lists a bunch more packages that are [required for install](https://github.com/iterative/tpi/blob/cc2683e8f4a45bb451f574c860df3b651bfd1c9a/setup.cfg#L62-L67). Any reason those are not listed, but instead only the optional `sshfs` is listed?

Reply:
quite right - just added them.